### ISSUE TITLE
Some problems with colapsed component group

### ISSUE THREAD

**Issue by qk4l** [CachetHQ/Cachet#1683]
Hello, 

I notice that it's not possible to create non collapsed component group on you demo site.
![cachet_2 2_groups_collapsed](https://cloud.githubusercontent.com/assets/8394059/14227765/0922e3ea-f916-11e5-8162-888a8fcd03f0.png)

Additionally `collapsed` field do not have default value #1402 if it creates via API and this makes it required.
```
postgres_1 | ERROR:  null value in column "collapsed" violates not-null constraint
postgres_1 | DETAIL:  Failing row contains (22, test, 2016-04-02 17:14:58, 2016-04-02 17:14:58, 0, null).
postgres_1 | STATEMENT:  insert into "component_groups" ("name", "order", "collapsed", "updated_at", "created_at") values ($1, $2, $3, $4, $5) returning "id"
```

**Comment by GrahamCampbell** [comment]
I think our docs are wrong. Try `0` instead of `false`.

**Comment by GrahamCampbell** [comment]
We only accept the values `0`, `1`, or `2`.

**Comment by GrahamCampbell** [comment]
https://github.com/CachetHQ/Cachet/blob/master/app/Bus/Commands/ComponentGroup/AddComponentGroupCommand.php#L50

**Comment by jbrooksuk** [comment]
Docs need updating.

**Comment by jbrooksuk** [comment]
Docs are fixed, sorry about that!

**Comment by qk4l** [comment]
@jbrooksuk Please mark `collapsed` field as required.  Thank you.

**Comment by jbrooksuk** [comment]
Should default to `0`, no?

**Comment by qk4l** [comment]
Think so, but now if you do not provide this filed you would got SQL error.

*Query*
```
$  curl -X POST -d name=test2   --header "X-Cachet-Token: TOKEN" localhost/api/v1/components/groups
{"errors":[{"id":"38d67aa0-0d4f-437c-8eaf-8d7abde98c47","status":400,"title":"Bad Request","detail":"The request cannot be fulfilled due to bad syntax."}]}
```
*SQL Error*
```
postgres_1 | ERROR:  null value in column "collapsed" violates not-null constraint
postgres_1 | DETAIL:  Failing row contains (22, test, 2016-04-02 17:14:58, 2016-04-02 17:14:58, 0, null).
postgres_1 | STATEMENT:  insert into "component_groups" ("name", "order", "collapsed", "updated_at", "created_at") values ($1, $2, $3, $4, $5) returning "id"
```

**Comment by GrahamCampbell** [comment]
Is our demo doing this?

**Comment by qk4l** [comment]
@GrahamCampbell  I don't have access to your demo DB to check it but for sure it's the same.
```
curl -X POST -d name=test2   --header "X-Cachet-Token: 9yMHsdioQosnyVK4iCVR" https://demo.cachethq.io/api/v1/components/groups -k

{"errors":[{"id":"35bb65dd-ca24-4a0f-bda4-9d804f2fc1d6","status":400,"title":"Bad Request","detail":"The request cannot be fulfilled due to bad syntax."}]}
```

**Comment by jbrooksuk** [comment]
Have you ran **php artisan app:update**?

**Comment by qk4l** [comment]
Does not help. I build Docker image from tag `v2.2.0` and got this error with fresh installed database right after initial setup.

**Comment by jbrooksuk** [comment]
This should be fixed.

**Comment by hubertgrzeskowiak** [comment]
Having the same problem with v2.2.2. Everything is okay if you provide the "collapsed" field but it yields an error if the field is missing. Tested with both JSON and form-encoding.


`truhgr@wkstru225 ~> curl -H "X-Cachet-Token: rCL9euWNbibLZqRWPEoN" -H "Content-Type: application/json" http://localhost/api/v1/components/groups -X POST -d '{"name":"services","order":0,"collapsed":0}'`

> 
{"data":{"order":0,"collapsed":0,"name":"services","updated_at":"2016-05-03 14:36:43","created_at":"2016-05-03 14:36:43","id":1,"lowest_human_status":null}}

`
truhgr@wkstru225 ~> curl -H "X-Cachet-Token: rCL9euWNbibLZqRWPEoN" -H "Content-Type: application/json" http://localhost/api/v1/components/groups -X POST -d '{"name":"services","order":0}'`

> 
{"errors":[{"id":"00ba3a68-199a-44d1-a012-8b06d696da0c","status":400,"title":"Bad Request","detail":"The request cannot be fulfilled due to bad syntax."}]}

**Comment by hubertgrzeskowiak** [comment]
Happens on the official API reference page too btw: https://docs.cachethq.io/docs/post-componentgroups

**Comment by jbrooksuk** [comment]
@hubertgrzeskowiak this will be fixed in v2.3.0 :)
